core/modules/modPrune.class.php

```diff
@@ -223,10 +223,9 @@ public function init($options = '')
 	 * @param   string  $options    Options when enabling module ('', 'noboxes')
 	 * @return  int                 1 if OK, 0 if KO
 	 */
-	public function remove($options = '')
-	{
+	public function remove($options= '') {
```

Comment:
**Reporter**: CHECKSTYLE
**Rule**: Squiz.Functions.MultiLineFunctionDeclaration.BraceOnSameLine
**Severity**: ERROR
**File**: core/modules/modPrune.class.php L226

Opening brace should be on a new line

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a961820172>*